app/controllers/orga/mailings_controller.rb

```diff
@@ -0,0 +1,63 @@
+class Orga::MailingsController < Orga::BaseController
+  before_action :normalize_params, only: [:create, :update]
+  before_action :find_mailing, only: [:show, :edit, :update, :destroy]
+
+  def index
+    authorize! :read, :mailing
+    @mailings = Mailing.order('id DESC').page(params[:page])
+  end
+
+  def show
+    authorize! :read, :mailing
+  end
+
+  def new
+    @mailing = Mailing.new
+  end
+
+  def create
+    @mailing = Mailing.new(mailing_params)
+    if @mailing.save
+      redirect_to orga_mailing_path(@mailing), notice: 'Mailing was successfully created.'
+    else
+      render action: :new
+    end
+  end
+
+  def update
```

Comment:
Why no `authorize!` for the update, edit and destroy actions?

Is it enough to know the user is a member of `orga`? If so, one could savely remove the `authorize!` calls for show and index as well.